lib_static/plugins/acts_as_journalized/lib/acts/journalized/journable_differ.rb

```diff
@@ -30,25 +30,29 @@ module Acts::Journalized
   class JournableDiffer
     class << self
       def changes(original, changed)
-        original_data = normalize_newlines(journaled_attributes(original))
+        original_data = original ? normalize_newlines(journaled_attributes(original)) : {}
 
         normalize_newlines(journaled_attributes(changed))
           .select { |attribute, new_value| no_nil_to_empty_strings?(original_data, attribute, new_value) }
           .to_h { |attribute, new_value| [attribute, [original_data[attribute], new_value]] }
           .with_indifferent_access
       end
 
+      def association_changes(original, changed, *association_params)
+        get_association_changes(original, changed, *association_params)
+      end
+
       private
 
       def normalize_newlines(data)
         data.each_with_object({}) do |e, h|
-          h[e[0]] = (e[1].is_a?(String) ? e[1].gsub(/\r\n/, "\n") : e[1])
+          h[e[0]] = (e[1].is_a?(String) ? e[1].gsub("\r\n", "\n") : e[1])
         end
       end
 
       def no_nil_to_empty_strings?(normalized_old_data, attribute, new_value)
         old_value = normalized_old_data[attribute]
-        new_value != old_value && (new_value.present? || old_value.present?)
+        new_value != old_value
```

Comment:
It is unsettling that no spec pics up on this. So I don't know how relevant my complaint is.

Reply:
I made the changes here 953bdf0f2ddf3e6d9dcb06b40ebc919fbd3cebb1. The expectations now fail in case any of the `nil -> ''` attribute transitions are included.
Some minor adjustment was required on the conditions, because `[false].any?` returns `false` and it is not safe to use it. I used `([new_value, old_value] - ['', nil]).present?` instead.